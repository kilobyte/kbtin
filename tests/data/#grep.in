#grep {.*} {} {#show ✓ empty} #else {#show ✗ empty}
#grep {x.*} {} {#show ✓ empty} #else {#show ✗ empty}
#grep {foo.*bar} {fooquuxbar} {#show ✓ foobar} #else {#show ✗ foobar}
#grep {foo.*bar} {fooquuxbaz} {#show ✓ foobaz} #else {#show ✗ foobaz}
#grep foo bar {#show foo=bar} #elif {#grep foo bar} {#show inline ✓ foo=bar} #else {#show inline ✗ foo=bar}
#grep foo bar {#show foo=bar} #elif {#grep foo foo} {#show inline ✓ foo=foo} #else {#show inline ✗ foo=foo}
