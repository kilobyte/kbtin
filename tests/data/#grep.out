✓ empty
✗ empty
✓ foobar
✗ foobaz
inline ✗ foo=bar
inline ✓ foo=foo
